src/parser/dep.fs

```diff
@@ -330,8 +330,9 @@ let collect_one (verify_flags: list<(string * ref<bool>)>) (verify_mode: verify_
     | Pragma _ ->
         ()
     | TopLevelModule lid ->
-        raise (Err (Util.format1 "Automatic dependency analysis demands one \
-          module per file (module %s not supported)" (string_of_lid lid true)))
+        ()
+//        raise (Err (Util.format1 "Automatic dependency analysis demands one \
+//          module per file (module %s not supported)" (string_of_lid lid true)))
```

Comment:
In dep.collect_file, I can replace the Warning in the non-singleton case by an Err instead. That should what we want with minimum change.